setup.py

```diff
@@ -68,12 +68,12 @@
 
 __entry_points__ = {
     'gpiozero_pin_factories': [
-        'PiGPIOPin  = gpiozero.pins.pigpiod:PiGPIOPin',
-        'RPiGPIOPin = gpiozero.pins.rpigpio:RPiGPIOPin',
-        'RPIOPin    = gpiozero.pins.rpio:RPIOPin',
-        'NativePin  = gpiozero.pins.native:NativePin',
-        'MockPin    = gpiozero.pins.mock:MockPin',
-        'MockPWMPin = gpiozero.pins.mock:MockPWMPin',
+        'pigpio  = gpiozero.pins.pigpiod:PiGPIOFactory',
+        'rpigpio = gpiozero.pins.rpigpio:RPiGPIOFactory',
+        'rpio    = gpiozero.pins.rpio:RPIOFactory',
+        'native  = gpiozero.pins.native:NativeFactory',
+        'mock    = gpiozero.pins.mock:MockFactory',
+        'mockpwm = gpiozero.pins.mock:MockPWMFactory',
```

Comment:
If that's a common use-case I'd be more tempted to make the pin-class configurable from the command line as it's more generic and hence will cover more use-cases with a single change (under the assumption we're bound to add more pin classes for things like analog etc. and I'd rather avoid adding yet more mock factory entry points)